### DISCUSSION TITLE
Why is the Shortcuts pane hidden instead of removed?

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#1768] [Q&A] [answered]
I'm asking because I've been seeing it randomly show up since after 1.37.2 (I think).

I originally thought it was reintroduced, but then it'd disappear. And show up again.
I remember testing downgrading uBO to 1.37.2 and then to 1.37 and 1.36.2 respectively to see if it'd get removed, but that didn't change anything. <= If I closed the Dashboard without switching between any of the panes, the Shortcuts pane would always be there after reopening the Dashboard (even after a browser restart). It would always (?) disappear after switching panes and restarting the browser, though.

This didn't seem issue-worthy to me, especially since I couldn't find the steps to make the pane show up again (like I mentioned, it'd show up randomly).

I haven't seen it show up since the introduction of the Support pane, but I did just see it on the latest stable on one of my other FF profiles, so I figured I'd ask.

Here's a screenshot from that profile: https://i.imgur.com/f1rozJy.png
And from my main one: https://i.imgur.com/khxE9h7.png

So what causes  `body` `.canUpdateShortcuts` / why not remove the pane?

**Comment by uBlock-user** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/6836809aad965cdf0f923558df95d263dd48ed02

https://github.com/uBlockOrigin/uBlock-issues/issues/1142

Still needed for FF < 74

**Reply by u-RraaLL** [reply]
Oh, I see. I don't know why I assumed older FF can't have latest uBO installed...

Anyway, nice to see it's fixed. Why is it not mentioned in the changelog for the dev releases though?

**Reply by uBlock-user** [reply]
@gorhill

**Reply by gorhill** [reply]
I try to keep the release notes for things which are of interest for end users -- I assumed this glitch was mostly not known by people, thus there was no point bringing it up.
